### ISSUE TITLE
How to set `find_unused_parameters=True`?

### ISSUE THREAD

**Issue by chenhaiwen** [facebookresearch/detectron2#4191]
When I train with multiple Gpus, have a RuntimeError:

 _Expected to have finished reduction in the prior iteration before starting a new one. This error indicates that your module has parameters that were not used in producing loss. You can enable unused parameter detection by (1) **passing the keyword argument `find_unused_parameters=True` to `torch.nn.parallel.DistributedDataParallel`;** (2) making sure all `forward` function outputs participate in calculating loss. If you already have done the above two steps, then the distributed data parallel module wasn't able to locate the output tensors in the return value of your module's `forward` function. Please include the loss function and the structure of the return value of `forward` of your module when reporting this issue (e.g. list, dict, iterable)._
